python/perf-kernels/flash-attention.py

```diff
@@ -259,12 +366,12 @@ def _attn_fwd(
     # initialize offsets
     offs_m = start_m * BLOCK_M + tl.arange(0, BLOCK_M)
     offs_n = tl.arange(0, BLOCK_N)
-    if bias is not None:
+    if BIAS_TYPE is not None:
```

Comment:
Ideally `BIAS_TYPE` should be a integer rather than a string.
I haven't figured out how to compile a triton kernel with string type `tl.constexpr`.

Can be done later.

Reply:
Fixed.